common/lib/xmodule/xmodule/graders.py

```diff
@@ -429,11 +429,13 @@ def grade(self, grade_sheet, generate_random_scores=False):
             breakdown = [{'percent': total_percent, 'label': total_label,
                           'detail': total_detail, 'category': self.category, 'prominent': True}, ]
         else:
-            total_detail = u"{section_type} Average = {percent:.0%}".format(
+            # Translators: "Homework Average = 0%"
+            total_detail = _(u"{section_type} Average = {percent:.0%}").format(
```

Comment:
Sanity check: Does the i18n_tool validator properly handle these format arguments?

Reply:
Yes.